src/relentless/simulate/dilute.py

```diff
@@ -208,7 +213,12 @@ def process(self, sim, sim_op):
         # adjust extent to maintain pressure if barostat
         else:
             coeffs = numpy.array([-ens.P / kT, 1, B])
-            v = numpy.max(numpy.roots(coeffs))
+            roots = numpy.roots(coeffs)
+            if not numpy.allclose(numpy.imag(roots), 0):
+                raise ValueError(
+                    "Unable to solve for volume, imaginary roots obtained."
+                )
+            v = numpy.max(numpy.real(roots))
```

Comment:
@clpetix I was getting a warning in my unit tests related to casting imaginary values to real, so I added this check here. I think the choice of potential parameters + pressure in the NPT dilute test may not be solvable. Could you please investigate and, if that's all that is the issue, let me know what an appropriate choice would be?

Reply:
@clpetix I modified this test so that now it uses NVT to get the pressure for known V, then does NPT to get V back for the same P. If you have a spreadsheet somewhere, could you compute what the P that comes from the NVT simulation is supposed to be? I think it was `1.25`

Reply:
@mphoward the answer is 1.25 for this step size and cut-off of r!